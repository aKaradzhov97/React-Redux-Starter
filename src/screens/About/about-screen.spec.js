import React from 'react';
import { render, screen } from '../../tests/test-utils';
import AboutScreen from './about-screen';

describe('AboutScreen', () => {
  let initialState;

  beforeAll(() => {
    initialState = {};
  });

  describe('Rendering', () => {
    it('should render the component with initial state', () => {
      const { container } = render(<AboutScreen />, { initialState });

      expect(container.querySelector('body section')).toMatchSnapshot();
    });
  });
});
