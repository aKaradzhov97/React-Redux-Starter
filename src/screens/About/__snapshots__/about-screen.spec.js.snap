// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutScreen Rendering should render the component with initial state 1`] = `
<section>
  <h1>
    about_screen.title
  </h1>
  <a
    href="/"
  >
    about_screen.go_home_link
  </a>
</section>
`;
